src/fix_calibration.R

```diff
@@ -0,0 +1,389 @@
+#' @title Correct data with underlying bad calibrations using In-Situ calibration reports
+#'
+#' @description This function uses calibration reports collected by field crews to "correct"
+#' instances of poor calibration. These are infrequent, and this function is not yet complete.
+#'
+#' @param df Site-parameter data frame
+#' @param cal_errors A data frame containing a list of known instances of poor calibration
+#'
+#' @return A data frame with modified `mean` values corrected for improper calibration. These observations are also identified in the `cal_fix` column.
+#'
+#' @examples
+#' add_range_flags(df = all_data_flagged$`archery-Actual Conductivity`)
+#' add_range_flags(df = all_data_flagged$`boxelder-Temperature`)
+
+fix_calibration <- function(df, cal_errors = readxl::read_excel("data/calibration_error_log.xlsx")){
```

Comment:
Note: I am realizing that we have a couple of look up tables that are being updated manually still that the FC system will need. Tagging this so that we think of a system to update those easily within the FC system.